src/test/regress/expected/multi_transactional_drop_shards.out

```diff
@@ -379,10 +387,10 @@ ORDER BY
 -- verify shards are not dropped
 \c - - - :worker_1_port
 \dt transactional_drop_reference*
-                        List of relations
- Schema |                 Name                 | Type  |  Owner
+                    List of relations
```

Comment:
I guess we expected to show the shard here but now the test shows the table itself?

Reply:
I added `_` so it shows shards again